setup.cfg

```diff
@@ -86,6 +86,7 @@ nova.scheduler.driver =
     caching_scheduler = nova.scheduler.caching_scheduler:CachingScheduler
     chance_scheduler = nova.scheduler.chance:ChanceScheduler
     fake_scheduler = nova.tests.unit.scheduler.fakes:FakeScheduler
+    turbonomic_scheduler = nova.scheduler.turbonomic_scheduler:TurbonomicScheduler
```

Comment:
We should not commit this.

This should be enough to work with the Turbonomic scheduler.
 2) Add turbonomic_driver to <Python 2.7>/site-packages/nova-16.1.0-py2.7.egg-info/entry_points.txt